.buildkite/scripts/pipelines/pull_request/pipeline.ts

```diff
@@ -28,21 +32,16 @@ const getPipeline = (filename: string, removeSteps = true) => {
 };
 
 (async () => {
+  const pipeline: string[] = [];
+
   try {
     const skippable = await areChangesSkippable(SKIPPABLE_PR_MATCHERS, REQUIRED_PATHS);
 
     if (skippable) {
-      console.log('All changes in PR are skippable. Skipping CI.');
-
-      // Since we skip everything, including post-build, we need to at least make sure the commit status gets set
-      execSync('BUILD_SUCCESSFUL=true .buildkite/scripts/lifecycle/commit_status_complete.sh', {
```

Comment:
This looks like it's still needed.

Reply:
<img width="878" alt="Screenshot 2024-07-20 at 12 16 56" src="https://github.com/user-attachments/assets/aac640e8-715c-42f2-9dfc-36378dd3be48">
It looks like the check is passing, so if this is meant to force-tick the kibana-ci check, it's not needed